builder/proxmox/common/step_start_vm.go

```diff
@@ -119,6 +119,7 @@ func (s *stepStartVM) Run(ctx context.Context, state multistep.StateBag) multist
 		Memory:       c.Memory,
 		QemuCores:    c.Cores,
 		QemuSockets:  c.Sockets,
+		QemuNuma:     &c.Numa,
```

Comment:
Is there a reason why this is a `*bool`? The rest of the arguments look like they're passed by copy so I wonder why this one in particular is different.

Reply:
That's what `proxmox-api-go` does - we use their `ConfigQemu` struct directly.
They do this to have a way to differentiate between `nil` and `false`. The Proxmox-API is sometimes tricky to work with, since setting something to `false` (or `0` for integers) often means something different than the default. Having a `nil` pointer is usually the `proxmox-api-go` way of saying "attribute is unset, use the default".
See here for some details on this matter: https://github.com/hashicorp/packer-plugin-proxmox/issues/127#issuecomment-1464030102

Reply:
I see, in this case it seems that it will never be `nil` since the attribute is a non-pointer boolean in the configs, so it will always be a pointer to either `true` or `false` (default).
I presume that it shouldn't be a problem though, it just means that we'll never have the default (unspecified) value anymore for this attribute.
Is this correct?